starter_code/queries.md

```diff
@@ -4,80 +4,80 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+> db.companies.find({name:"Babelgum"}, {_id:0, name:1}).pretty()
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+> db.companies.find( {number_of_employees: {$gt:5000}}, {_id:0, name:1, number_of_employees:1}).sort({number_of_employees:-1}).limit(20);
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+> db.companies.find( { $and: [{founded_year:{$gte:2000}}, {founded_year:{$lte:2005}} ] }, {_id:0, name:1, founded_year:1}).sort({founded_year:1})
 
 ### 4. All the companies that had a Valuation Amount of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
 
-<!-- Your Code Goes Here -->
+> db.companies.find({founded_year:{$lt:2010},'ipo.valuation_amount':{$gt:100000000}}, {_id:0, name:1, ipo:1}).sort({'ipo.valuation_amount':1}).pretty()
```

Comment:
Não era necessário ordenar pelo 'valuation_amount'. Bom uso da busca com o 'and' implícito! Parabéns.